From d5d12dc56c5ca35c99f03351b390c605db3d1d3e Mon Sep 17 00:00:00 2001
From: Angela Fong <angela.fong@joyent.com>
Date: Wed, 14 Feb 2018 15:38:00 -0800
Subject: [PATCH] ADMINUI-2398 CN Setup always shows 'Setup Failed' during
 setup Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 sapi_manifests/adminui/template | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sapi_manifests/adminui/template b/sapi_manifests/adminui/template
index b46d43a2..74a02725 100644
--- a/sapi_manifests/adminui/template
+++ b/sapi_manifests/adminui/template
@@ -77,5 +77,5 @@
 			}
 		}
 	},
-	"serverSetupJobName": "server-setup-1.0.2"
+	"serverSetupJobName": "server-setup-*"
 }
-- 
2.21.0

